components/dialog/index.css

```diff
@@ -11,377 +11,429 @@ governing permissions and limitations under the License.
+	--spectrum-standard-dialog-description-font-size: var(--spectrum-standard-dialog-body-font-size);
+	--spectrum-standard-dialog-description-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-standard-dialog-description-font-family: var(--spectrum-body-sans-serif-font-family);
+
+	--spectrum-standard-dialog-heading-line-height: var(--spectrum-title-line-height);
+	--spectrum-standard-dialog-description-line-height: var(--spectrum-line-height-200);
+
+	/* Colors */
+	--spectrum-standard-dialog-heading-text-color: var(--spectrum-heading-color);
+	--spectrum-standard-dialog-description-text-color: var(--spectrum-body-color);
+	--spectrum-standard-dialog-background-color: var(--spectrum-background-layer-2-color);
+
+	/* Spacing */
+	--spectrum-standard-dialog-spacing-edge-to-content: var(--spectrum-spacing-500);
+	--spectrum-standard-dialog-spacing-title-to-description: var(--spectrum-spacing-300);
+	--spectrum-standard-dialog-spacing-description-to-footer: var(--spectrum-spacing-500);
+	--spectrum-standard-dialog-spacing-title-to-header-content: var(--spectrum-spacing-400);
+	--spectrum-standard-dialog-spacing-edge-to-header-content-inline-end: var(--spectrum-spacing-400);
+
+	/* Grid */
+	--spectrum-standard-dialog-spacing-grid-padding: var(--spectrum-standard-dialog-spacing-edge-to-content);
+	--spectrum-standard-dialog-spacing-edge-to-close-button: var(--spectrum-spacing-200);
+
+	/* TODO: is there a default token somewhere for this hero height? */
+	/* Hero */
+	--spectrum-standard-dialog-min-hero-block-size: 140px;
 }
 
 .spectrum-Dialog {
-  /* Be a flexbox to allow a full sized content area that scrolls */
-  display: flex;
-
-  /* Allow 100% width, taking into account padding */
-  box-sizing: border-box;
-
-  /* Be no bigger than max-width,  but also be 90% if the viewport is smaller than max-width */
-  inline-size: fit-content;
-  min-inline-size: var(--mod-dialog-min-inline-size, var(--spectrum-dialog-min-inline-size));
-  max-inline-size: 100%;
-
-  max-block-size: inherit;
-
-  outline: none;
-}
-
-.spectrum-Dialog--small {
-  inline-size: var(--mod-dialog-confirm-small-width, var(--spectrum-dialog-confirm-small-width));
+	/* Be a flexbox to allow a full sized content area that scrolls */
+	display: flex;
+
+	/* Allow 100% width, taking into account padding */
+	box-sizing: border-box;
+
+	/* Be no bigger than max-width,  but also be 90% if the viewport is smaller than max-width */
+	inline-size: var(--mod-standard-dialog-max-width, var(--spectrum-standard-dialog-max-width));
+	min-inline-size: var(--mod-standard-dialog-min-inline-size, var(--spectrum-standard-dialog-min-inline-size));
+	max-inline-size: 100%;
+	max-block-size: inherit;
+	background-color: var(--spectrum-standard-dialog-background-color);
+	outline: none;
+	border-radius: var(--mod-standard-dialog-border-radius, var(--spectrum-standard-dialog-border-radius));
```

Comment:
I didn't initially have the `border-radius` set here- I set it in `Modal`. But if I don't set it here, the Chromatic preview gets a little messy, since they can't be wrapped in a `Modal`. What's the best way to handle that? Does this work? There's definitely a space between the dialog border radius and the Modal border radius, especially if you're in WHCM.

<img width="1018" alt="Screenshot 2024-06-24 at 2 08 40 PM" src="https://github.com/adobe/spectrum-css/assets/69602589/72222478-6354-4b99-b8e4-3607a682cbff">

Reply:
I think this is happening because the inner border radius on the border isn't the same as the outer border radius, so we need to adjust the border-radius either of the hero image or on the border. I believe this might just be a [straightforward addition or subtraction of the border width](https://www.js-craft.io/blog/css-inner-outer-border-radius/#what-is-the-correct-ratio-between-the-inner-and-outer-borderradius), so with the border width being 3px (I don't know what token exactly that's coming from but I can see in the inspector it's 3px), either subtract it from the border-radius on the hero, or add it to the border-radius for forced-colors (this seems like the better option to me), maybe something like this:
![image](https://github.com/adobe/spectrum-css/assets/54716846/3fce138f-9452-4012-8738-dca426f4ab45)

Reply:
Ok I get what you're talking about. But I don't see the 3px border width specifically. The forced colors query is only declaring `border`, so shouldn't it be 1px? There's not a width for it, unless it's somewhere else? What am I not looking at or finding? 

What if I changed around where the `overflow: hidden` was? Right now it's on the `--hero`, but if I moved `overflow: hidden` to `.spectrum-Dialog` instead, it gets rid of the need for any of the border radius calculations. I think it does what we are trying to create here, but I'm not sure why it was on `--hero` in the first place. Would this affect anything negatively? Things are still scrolling as expected even with the overflow: hidden on the dialog now.

https://github.com/adobe/spectrum-css/pull/2860/commits/a1ef47e07ebfa9ce759d45a89416ae992a33ab87 (if we like this approach I need to clean up the comments)